src/modules/0.0.4/iasql_functions/sql/create_fns.sql

```diff
@@ -1,3 +1,22 @@
+-- TODO: Does this belong here or in a similar file in the iasql_platform module?
+create or replace function iasql_audit() returns trigger
+language plpgsql security definer
+as $$
+begin
+  if (TG_OP = 'INSERT') then
```

Comment:
does the `TG_OP` is a Postgres standard to define the trigger operation or where does it come from?